docs/Documentation/Frequently asked questions.md

```diff
@@ -122,5 +138,5 @@ IoTDB> show timeseries root.fit.d1.*
 The default IoTDB's CLI time display format is human readable (e.g. ```1970-01-01T08:00:00.001```), if you want to display time in timestamp type or other readable format, add parameter ```-disableIS08601``` in start command:
 
 ```
->sh IOTDB_HOME/iotdb/bin/start-client.sh -h 10.129.187.21 -p 6667 -u root -pw root -disableIS08601
-```
\ No newline at end of file
+> sh /workspace/incubator-iotdb/iotdb-cli/cli/bin/start-client.sh -h 127.0.0.1 -p 6667 -u root -pw root -disableIS08601
```

Comment:
This is better than `cd /workspace/incubator-iotdb/iotdb-cli/cli/bin && ./start-client.sh ...`